api/Notification.json

```diff
@@ -13,10 +13,11 @@
             ]
           },
           "chrome_android": {
-            "version_added": "25",
+            "version_added": "42",
+            "partial_implementation": true,
             "notes": [
-              "Starting in Chrome 49, notifications do not work in incognito mode.",
-              "Before Chrome 42, service worker additions were not supported."
+              "Notifications in Chrome for Android are only available through service workers. Developers must use <a href='https://developer.mozilla.org/docs/Web/API/ServiceWorkerRegistration/showNotification'><code>ServiceWorkerRegistration.showNotification()</code></a> instead.",
```

Comment:
The "Developers must use" style seems unusual in BCD. Is it copied from somewhere, and if so can all occurrences be fixed to just speak directly to the developer instead of about them?